input.lisp

```diff
@@ -420,58 +425,58 @@ match with an element of the completions."
 
       ;; Display the input window text.
       (loop for i from 0 below (+ prompt-lines-length completions-length)
-            if (< i prompt-lines-length)
-              do (draw-image-glyphs win gcontext font
-                                    *message-window-padding*
-                                    (prompt-text-y i font *message-window-y-padding*)
-                                    (nth i prompt-lines)
-                                    :translate #'translate-id
-                                    :size 16)
-            else
-              do (draw-image-glyphs win gcontext font
-                                    *message-window-padding*
-                                    (prompt-text-y i font *message-window-y-padding*)
-                                    (nth (- i prompt-lines-length) completions)
-                                    :translate #'translate-id
-                                    :size 16))
+         if (< i prompt-lines-length)
+         do (draw-image-glyphs win gcontext font
+                               *message-window-padding*
+                               (prompt-text-y i font *message-window-y-padding*)
+                               (nth i prompt-lines)
+                               :translate #'translate-id
+                               :size 16)
+         else
+         do (draw-image-glyphs win gcontext font
+                               *message-window-padding*
+                               (prompt-text-y i font *message-window-y-padding*)
+                               (nth (- i prompt-lines-length) completions)
+                               :translate #'translate-id
+                               :size 16))
```

Comment:
Auto-formatting from my emacs. Sorry about this. The next block of "changes" is the same.